pkg/spec3/benchmark_serialization_test.go

```diff
@@ -19,6 +20,29 @@ var swaggerDiffOptions = []cmp.Option{cmp.Comparer(func(a spec.Ref, b spec.Ref)
 	return a.String() == b.String()
 })}
 
+func TestOpenAPIV3RoundTrip(t *testing.T) {
```

Comment:
https://github.com/kubernetes/kube-openapi/blob/f73e7dbcfe2954706d8ee5db7f1adaed3950e8de/pkg/util/jsontesting/json_roundtrip.go#L28-L44

might be suitable to use here

Reply:
To conform to the interface, Unmarshal is implemented in https://github.com/kubernetes/kube-openapi/pull/368, can I fix this in a follow up?